diem-move/diem-framework/experimental/sources/NFTGallery.move

```diff
@@ -0,0 +1,122 @@
+module 0x1::NFTGallery {
+    use Std::Errors;
+    use Std::GUID;
+    use 0x1::NFT::{Self, Token};
+    use Std::Option::{Self, Option};
+    use Std::Signer;
+    use Std::Vector;
+
+    /// Gallery holding tokens of `TokenType` as well as information of approved operators.
+    struct NFTGallery<phantom TokenType: store> has key {
+        gallery: vector<Token<TokenType>>
+    }
+
+    const MAX_U64: u128 = 18446744073709551615u128;
```

Comment:
I think I commented this on someone else's PR (not necessarily yours to fix).  We should probably just have this built into the language.

Reply:
I think it's actually not used here at all, remove?